README.md

```diff
@@ -410,15 +410,15 @@ Custom configuration for the Celery workers are listed below:
 * `cachito_nexus_hoster_password` - the password of the Nexus service account used by Cachito for
   the Nexus instance that has the hosted repositories. This is used instead of
   `cachito_nexus_password` for uploading content if you are using the two Nexus instance approach as
-  described in the "Nexus For npm" section. If this is set, `cachito_nexus_hoster_username` must
+  described in the "Nexus For Java Script" section. If this is set, `cachito_nexus_hoster_username` must
```

Comment:
On a side note, the description of two nexus approaches should be moved to a general section since it is and will be used for more package managers.